tests/reference/llvm-expr_01-54467c1.stdout

```diff
@@ -5,16 +5,16 @@ source_filename = "LFortran"
 @1 = private unnamed_addr constant [2 x i8] c"\0A\00", align 1
 @2 = private unnamed_addr constant [5 x i8] c"%d%s\00", align 1
 
-define void @__module__global_symbols__lpython_main_program() {
+define void @__module___main_____main____global_statements() {
```

Comment:
We need to handle this name mangling correctly.

Reply:
Got it. I guess we can work on it in a separate PR.